src/main/java/InputData/playlistInputData.java

```diff
@@ -1,4 +1,32 @@
 package InputData;
 
+import Entities.Song;
+
+import java.util.ArrayList;
```

Comment:
Note the change from ArrayList -> LinkedList within Playlist class

Reply:
Yep, at Rohan's request. Doesn't really matter which iterable we use up until it gets into a Playlist object anyways.

Reply:
I figured, thought I'd point it out just in case though